packages/back-end/src/queryRunners/PastExperimentsQueryRunner.ts

```diff
@@ -43,6 +43,10 @@ export class PastExperimentsQueryRunner extends QueryRunner<
         process: (rows) =>
           this.processPastExperimentQueryResponse(rows, merge, params.from),
         queryType: "pastExperiment",
+        querySource: {
+          sourceType: "PastExperiments",
+          id: this.model.id,
+        },
```

Comment:
Same question as for DimensionSlices. It seems like a PastExperiments import doesn't have any canonical point to link back to but it might still be worth keeping the source tagged